test/unit/table_report_history_test.sql

```diff
@@ -109,9 +109,9 @@ select has_pk('swrs_history', 'report', 'ggircs_report has primary key');
 select isnt_empty('select * from swrs_history.report', 'there is data in swrs.report');
 
 -- Report has correct emission total data
-select results_eq(
```

Comment:
I did this instead of adding `order by` to the materialized view because you can't replace or update materialized views, only drop them, and then you have to drop and recreate everything that references that view too